libs/pando/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "module": "commonjs",
+    "module": "ES2022",
```

Comment:
@lantua is this bad

Reply:
if it compiles (and test succeed), should be fine